backend/src/v4/routes/maps.js

```diff
@@ -596,9 +531,9 @@ function getOSMTile(req, res) {
 }
 
 function getHereBaseInfo(req, res) {
-	const domain = "1" + hereBaseDomain;
-	let uri = "/maptile/2.1/info";
-	uri += "?apiKey=" + config.here.apiKey;
+	const domain = hereBaseDomain;
+	let uri = "/v3/info";
```

Comment:
Not really sure why we need this to be broken up into 2 lines?

```suggestion
	let uri = `/v3/info?apiKey=${config.here.apiKey}`;
```